lib/main.gi

```diff
@@ -1393,7 +1393,7 @@ else
                      )
                 ];
     fi;
-  return A; #GlobalSplittingOfCyclotomicAlgebra(A);
+  return GlobalSplittingOfCyclotomicAlgebra(A);
```

Comment:
I find "re-add" in commit message confusing. Furthermore, `Global Splitting` will obstruct search for `GlobalSplittingOfCyclotomicAlgebra` in commit messages. Perhaps you want to write
```
Modify output of SimpleAlgebraInfoByData

It now returns GlobalSplittingOfCyclotomicAlgebra(A) instead of A.
```